setup.py

```diff
@@ -23,7 +23,7 @@
     },
     entry_points={
         'pytest11': [
-            'pytest_fixtures=pytest_fixtures'
+            'pytest_fixtures=nameko_sqlalchemy.pytest_fixtures'
```

Comment:
While we're making changes, I think the "keyword" here is how this plugin is listed when you run `py.test --trace-config`. It would be more clear if we used "nameko_sqlalchemy" instead of "pytest_fixtures".